packages/expo-modules-core/ios/Core/Views/ViewModuleWrapper.swift

```diff
@@ -57,7 +57,8 @@ public final class ViewModuleWrapper: RCTViewManager, DynamicModuleWrapperProtoc
    */
   @objc
   public func name() -> String {
-    return wrappedModuleHolder.name
+    // swiftlint:disable:next force_unwrapping
+    return wrappedModuleHolder!.name
```

Comment:
```suggestion
    guard let moduleHolder = wrappedModuleHolder else {
      fatalError("ModuleHolder is lost")
    }
    return moduleHolder.name
```

maybe we should better remove the force unwrap. feel free to fine tune the error message